src/hera.cpp

```diff
@@ -367,40 +387,12 @@ int hera_set_option(
   char const *value
 ) noexcept {
   hera_instance* hera = static_cast<hera_instance*>(instance);
-  if (strcmp(name, "fallback") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::fallback;
-    return 1;
-  }
-
-  if (strcmp(name, "evm2wasm") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::evm2wasm_contract;
-    return 1;
-  }
-
-  if (strcmp(name, "evm2wasm.cpp") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::evm2wasm_cpp;
-    return 1;
-  }
-
-  if (strcmp(name, "evm2wasm.cpp-trace") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::evm2wasm_cpp_tracing;
-    return 1;
-  }
-
-  if (strcmp(name, "evm2wasm.js") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::evm2wasm_js;
-    return 1;
-  }
 
-  if (strcmp(name, "evm2wasm.js-trace") == 0) {
-    if (strcmp(value, "true") == 0)
-      hera->evm_mode = hera_evm_mode::evm2wasm_js_tracing;
-    return 1;
+  if (strcmp(name, "evm") == 0) {
```

Comment:
This is somehow weird pattern, you can 3 comparisons instead of 1. Try
```
auto it = evm_mode_options.find(name)
if (it != evm_mode_options.end())
    hera->evm_mode = it->second;
```

You should handle all cases with this single snippet. I have not tested it.

Reply:
I actually want it to have a single key `evm`. The previous commit practically did what you suggest.